src/librustdoc/html/markdown.rs

```diff
@@ -78,6 +80,7 @@ pub struct Markdown<'a>(
     /// Default edition to use when parsing doctests (to add a `fn main`).
     pub Edition,
     pub &'a Option<Playground>,
+    pub u32,
```

Comment:
This is getting silly. A struct with seven member variables should just be a regular struct, with named fields, not a tuple.

Reply:
Also, `u32` could be replaced with `u8` since it never goes above 6 in principle... (just nitpicking)